src/anyp/Uri.cc

```diff
@@ -219,37 +203,37 @@ AnyP::Uri::parse(const HttpRequestMethod& method, const char *url)
          */
         foundPort = 443;
 
+        // XXX: use tokenizer
```

Comment:
I think this XXX (and a performance regression?) should be addressed in this PR. There are only two sscanf() calls that need to be replaced, and we probably already have the right APIs for extracting in-set characters and integers. Why skip this section?

Reply:
Just that I broke the work up into smaller chunks for easier and incremental review. Parsing the authority portion was to be the next step after scheme.